TMS.Net07.Homework2.DaysOfWeek/TMS.Net07.Homework2.DaysOfWeek/Program.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Threading;
+
+namespace TMS.Net07.Homework2.DaysOfWeek
+{
+    enum DayOfWeek : byte
+    {
+        Sunday,
+        Monday,
+        Tuesday,
+        Wednesday,
+        Thursday,
+        Friday,
+        Saturday
+    }
+
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            while (true)
+            {
+                Console.Write("Input a date before 2100: ");
+                string inputDate = Console.ReadLine(); // Inputted data
+
+                if (inputDate.ToLower() == "exit")
+                {
+                    Console.WriteLine();
+                    Console.Write("Exiting the program...");
+                    Thread.Sleep(1500);
+                    Environment.Exit(0);
```

Comment:
Avoid using `Environment.Exit(0);` - use `return` instead. `Environment.Exit(0);` closes the whole application instead of just finishing a method